### DISCUSSION TITLE
Configurable search placeholder

### DISCUSSION THREAD

**Discussion by margarizaldi** [filamentphp/filament#4104] [Ideas]
A very common use case is to tell users what they can search through the table or global search bar. Instead of the default "Search", developer should be able to customize like:
- `Search Invoice # or Shipping Number`
- `Search Product SKU`
- etc.

Maybe it can be a table method like `$table->searchPlaceholder('Lorem Ipsum')` or a resource variable `$searchPlaceholder = 'Lorem Ipsum';`

Hopefully someone will be happy to write a PR

**Comment by gp0** [comment]
For any looking for this: You can do this by placing a language file in `lang/vendor/filament-panels/en/global-search.php` and then override the placeholder there.

**Comment by raobilal482** [comment]
i do this like this
  ->searchPlaceholder(function (): string {
                $all_listing = Listing::isProperty()->count();
                return 'Search in '.$all_listing.' Properties';
            }
